app/controllers/work_packages/moves_controller.rb

```diff
@@ -81,32 +59,18 @@ def create
     end
       end
 
-  def set_flash_from_bulk_work_package_save(work_packages, unsaved_work_package_ids)
-    if unsaved_work_package_ids.empty? and not work_packages.empty?
+  def set_flash_from_bulk_work_package_save(work_packages, unsaved_work_packages)
+    if unsaved_work_packages.empty? and not work_packages.empty?
       flash[:notice] = (@copy) ? l(:notice_successful_create) : l(:notice_successful_update)
     else
       flash[:error] = l(:notice_failed_to_save_work_packages,
-                        count: unsaved_work_package_ids.size,
+                        count: unsaved_work_packages.size,
                         total: work_packages.size,
-                        ids: '#' + unsaved_work_package_ids.join(', #'))
+                        ids: unsaved_work_packages.map {|wp| '#' + wp.id}.join(', '))
```

Comment:
Line is too long. [85/80]<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.